build_docs.sh

```diff
@@ -45,7 +45,7 @@ bundle exec ruby plugindocs.rb --output-path ../logstash-docs ../logstash/plugin
 
 cd ../logstash
 
-perl ../docs/build_docs --respect_edit_url_overrides --doc docs/index.asciidoc --resource=../logstash-docs/docs/ --chunk 1
+perl ../docs/build_docs --asciidoctor --respect_edit_url_overrides --doc docs/index.asciidoc --resource=../logstash-docs/docs/ --chunk 1
```

Comment:
Remove the `perl` bit here, I think. You just want to run `build_docs`. Perl doesn't need to get involved. `build_docs` happens to be python but it has a proper shebang so we have full control.